### ISSUE TITLE
Batch dimension issue 

### ISSUE THREAD

**Issue by kyuheejo** [facebookresearch/dino#166]
I am getting the following dimension for images:

[ number of augmentations, batch, channel, w, h]

but I need: 

[ batch, number of augmentations, channel, w, h]

My data loader looks like this

def __getitem__(self, idx):
        if torch.is_tensor(idx):
            idx = idx.tolist()
        sample = self.samples.iloc[idx]
        img = Image.open(os.path.join(self.root, sample['image']))
        label = torch.tensor(sample['target'])
        if self.transform is not None:
            img = self.transform(img)
        return img, label

What is the problem??
